api/v1/installation_types.go

```diff
@@ -302,6 +303,34 @@ var (
 	ProviderTKG       Provider = "TKG"
 )
 
+func (p Provider) IsAKS() bool {
```

Comment:
We may as well also implement isProviderNone, since there is still a check in for ProviderNone in core_controller.go#L727

Reply:
Implement `IsNone()`.